index.bs

```diff
@@ -2651,30 +2662,44 @@ non-negative integer |defaultBudget|, and an [=aggregatable debug reporting conf
     with |value|, |budget|, |supportedTypes|, and |defaultConfig|.
 1. Return the [=tuple=] (|budget|, |config|).
 
-To <dfn>parse max event states</dfn> from a [=map=] |map| and
-a possibly null 32-bit positive integer |attributionScopeLimit|:
+To <dfn>parse attribution scope data</dfn> from a [=map=] |map|:
+1. If |map|["<code>[=source-registration JSON key/attribution_scope_data=]</code>"] does not [=map/exists|exist=], return null.
```

Comment:
What do you think about the following naming scheme, to avoid redundancy:

```jsonc
"attribution_scopes": {
  "values": [...], // was top-level attribution_scopes
  "limit": ...,    // was top-level attribution_scope_limit
  "max_event_states": ...
}
```

Reply:
+1

Reply:
Done. this indeed is much simpler. thanks!